flintrock/flintrock.py

```diff
@@ -334,6 +339,22 @@ def launch(
             )
         services += [spark]
 
+    # Format ec2 tags
```

Comment:
This should be split out into a function instead of being duplicated here and in `add_slaves()`. Then, that function should be used to validate the input right as it's parsed by Click:

http://click.pocoo.org/6/options/#callbacks-for-validation